_includes/community/com-sql-query-run.md

```diff
@@ -0,0 +1,9 @@
+| Method | Description | Further information |
+|---|---|---|
+| CLI | Run SQL statements from the FeatureBase CLI system | [Enable SQL CLI](/docs/community/com-config/com-config-sql-cli-enable) |
+| FeatureBase API | FeatureBase provides an API which can be used to execute SQL statements | [Community API](/docs/community/com-api/com-api-home) |
+| Data import and ingest | Recreate table structure based on CSV, SQL or Kafka source files | [Import data](/docs/community/com-ingest/com-ingest-manage) |
+| HTTPS | Run SQL statements over HTTPS using the SQL CLI | [Enable SQL CLI](/docs/community/com-config/com-config-sql-cli-enable)
```

Comment:
I think you have the wrong link as this goes to the CLI. https://docs.featurebase.com/docs/community/com-api/old-sql-endpoint/ is what you want

Reply:
fixerooni

Reply:
Doh! Thanks. Good catch.

> @ljsinclair I'm a little confused by the driving purpose of this PR. I think most of it made sense but db manage and table manage seem a little underdeveloped (especially when compared to cloud). Maybe you can walk me through this?

Three reasons:

1. There was no database page for Community which I thought would be useful for users (even if it's fairly brief in this iteration)
2. There's no tables page for Community either for same reason as above
3. I realised that the methods to create tables ran parallel to the methods of running SQL queries, and thus that's why the manage page has been reworked.